subsys/net/ip/trickle.c

```diff
@@ -21,6 +21,9 @@ LOG_MODULE_REGISTER(net_trickle, CONFIG_NET_TRICKLE_LOG_LEVEL);
 
```

Comment:
Please fix the commit message. The commit subject looks a bit weird, I do not get the "Debug" word here. Perhaps the commit subject could look like `net: trickle: Re-init trickle timer for multiple triggerings` or something like that.

Also the commit body could be more clearer, so something like
```
We need to re-initialize the trickle->timer delayed work to use
trickle_timeout() if there are multiple triggerings.

Fixes #15606 
```